examples/js/exporters/GLTFExporter.js

```diff
@@ -1119,8 +1130,20 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			if ( primitives.length === 0 ) {
+
+				return -1;
```

Comment:
In `processNode` we're returning `null` when no node is created. Would probably vote for `null` over `-1` personally but either way let's make it consistent.